dynamic_instrumentation/src/lib.rs

```diff
@@ -1,223 +0,0 @@
-#![feature(rustc_private)]
-extern crate rustc_ast;
-extern crate rustc_const_eval;
-extern crate rustc_data_structures;
-extern crate rustc_driver;
-extern crate rustc_index;
-extern crate rustc_interface;
-extern crate rustc_middle;
-extern crate rustc_mir_build;
-extern crate rustc_mir_transform;
-extern crate rustc_session;
-extern crate rustc_span;
-extern crate rustc_target;
-
-mod arg;
-mod hooks;
-mod instrument;
-mod into_operand;
-mod mir_utils;
-mod point;
-mod runtime_conversions;
-mod util;
-
-use instrument::Instrumenter;
-
-use cargo::core::compiler::{CompileMode, Context, DefaultExecutor, Executor, Unit};
-use cargo::core::{PackageId, Target, Verbosity, Workspace};
-use cargo::ops;
-use cargo::ops::CompileOptions;
-use cargo::util::important_paths::find_root_manifest_for_wd;
-use cargo::util::CargoResult;
-use cargo::Config;
-use cargo_util::ProcessBuilder;
-
-use rustc_ast::ast::{Item, ItemKind, Visibility, VisibilityKind};
-use rustc_ast::node_id::NodeId;
-use rustc_ast::ptr::P;
-use rustc_const_eval::transform::validate;
-use rustc_driver::Compilation;
-use rustc_interface::interface::Compiler;
-use rustc_interface::Queries;
-use rustc_middle::mir::MirPass;
-use rustc_middle::ty::query::{ExternProviders, Providers};
-use rustc_middle::ty::WithOptConstParam;
-use rustc_session::Session;
-use rustc_span::def_id::LocalDefId;
-use rustc_span::symbol::Ident;
-use rustc_span::DUMMY_SP;
-
-use anyhow::anyhow;
-use lazy_static::lazy_static;
-use std::env;
-use std::ffi::OsString;
-use std::path::Path;
-use std::sync::atomic::{AtomicBool, Ordering};
-use std::sync::{Arc, Mutex};
-
-lazy_static! {
-    /// TODO(kkysen) can be made non-lazy when `Mutex::new` is `const` in rust 1.63
-    static ref INSTRUMENTER: Instrumenter = Instrumenter::new();
-}
-
-struct NullCallbacks;
-
-impl rustc_driver::Callbacks for NullCallbacks {}
```

Comment:
This was deleted as now `TimePassesCallbacks::default()` is used instead, as that's `rustc`'s default (https://github.com/immunant/c2rust/pull/554/commits/8e9e94e7e1742983a38343925a1f847aff1a7ba9).